cleanup_utils/integration_cleanup.py

```diff
@@ -109,6 +109,8 @@ def run_integration_cleanup(dry_run=False):
         dry_run=dry_run
     )
     mysql_cleanup.run_cleanup()
+    if os.environ.get('DROP_INTEGRATION_DBS_AND_USERS', 'False').lower() == 'true':
+        mysql_cleanup.drop_integration_dbs_and_users()
```

Comment:
Uh-oh, somehow I missed this, but instead of using an IF here, could you pass a variable to mysql_cleanup to conditionally clean the users?
```
mysql_cleanup = MySqlCleanupInstance(
    age_limit=DEFAULT_AGE_LIMIT,
    url=os.environ['DEFAULT_INSTANCE_MYSQL_URL'],	        
    domain=os.environ['DEFAULT_INSTANCE_BASE_DOMAIN'],	
    drop_users=os.environ.get('DROP_INTEGRATION_DBS_AND_USERS', False),
    dry_run=dry_run
)
```

Reply:
Ah, good idea. Will do. Thanks.